### PR TITLE
Add links to links

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: content/home/docs.md [hunk 1]
```diff
@@ -99,4 +99,19 @@ subtitle = ""
...
 
+{{% col "col-12 col-sm-4" %}}
+<div class="featurette-icon">
+  <a href="https://seismo-learn.org/links/" target="_blank">
+    <i class="fas fa-bookmark fa-lg"></i>
+	<h4>地震“学”链接</h4>
+  </a>
+</div>
+
+🔖 地震学科研常用链接。
```
Comment (core-man) [inline content/home/docs.md:110]:
```suggestion
🔖 地震学科研常用链接~
```

File: content/home/docs.md [hunk 2]
```diff
@@ -99,4 +99,19 @@ subtitle = ""
...
+
+[<i class="fas fa-home fa-2x mr-2"></i>](https://seismo-learn.org/links/)
+[<i class="fab fa-github fa-2x mr-2"></i>](https://github.com/seismo-learn/links)
+[<i class="fas fa-bug fa-2x mr-2"></i>](https://github.com/seismo-learn/links/issues)
```
Comment (core-man) [inline content/home/docs.md:114]:
```suggestion
[<i class="fas fa-bug fa-2x mr-2"></i>](https://github.com/seismo-learn/links/issues)

[<i class="fas fa-comment fa-2x mr-2"></i>](https://github.com/seismo-learn/links/discussions)

```

Comment (seisman) [reply content/home/docs.md:114]: Do we really need discussion?

Comment (core-man) [reply content/home/docs.md:114]: not really for links

**Review by core-man — APPROVED** [review]

**Pull request merged**